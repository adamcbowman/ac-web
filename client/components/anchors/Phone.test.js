import React from 'react'
import Renderer from 'react-test-renderer'
import { Phone } from './index'

test('phone component', () => {
    let phone = Renderer.create(<Phone phone="250-837-2141" />)

    expect(phone).toMatchSnapshot()

    phone = Renderer.create(<Phone phone="250-837-2141" ext="112" />)

    expect(phone).toMatchSnapshot()

    phone = Renderer.create(
        <Phone phone="250-837-2141" ext="112">
            Call Karl Guillotte
        </Phone>
    )

    expect(phone).toMatchSnapshot()
})
