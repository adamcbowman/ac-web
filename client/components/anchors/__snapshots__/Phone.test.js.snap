// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`phone component 1`] = `
<a
  href="tel:250-837-2141"
>
  250-837-2141
</a>
`;

exports[`phone component 2`] = `
<a
  href="tel:250-837-2141"
>
  250-837-2141 ext. 112
</a>
`;

exports[`phone component 3`] = `
<a
  href="tel:250-837-2141"
>
  Call Karl Guillotte
</a>
`;
